server/src/internalClusterTest/java/org/opensearch/remotestore/ReplicaToPrimaryPromotionIT.java

```diff
@@ -122,12 +122,11 @@ public void testPromoteReplicaToPrimary() throws Exception {
         assertHitCount(client().prepareSearch(indexName).setSize(0).get(), numOfDocs);
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/opensearch-project/OpenSearch/issues/9130")
     public void testFailoverWhileIndexing() throws Exception {
         internalCluster().startNode();
         internalCluster().startNode();
         final String indexName = randomAlphaOfLength(5).toLowerCase(Locale.ROOT);
-        shard_count = scaledRandomIntBetween(1, 5);
+        shard_count = 1;
```

Comment:
no need to test with more number of shards?

Reply:
forgot to revert it . Reverted now.